### ISSUE TITLE
Konzept vLUG-Sprint

### ISSUE THREAD

**Issue by DebXWoody** [vlug-hessen/planung-und-ideen#7]
Erarbeiten und beschreiben des vLUG-Sprint Konzept.

**Comment by DebXWoody** [comment]
Auch wenn es hier sehr nette Funktionen für Diskussionen gibt, werden wir primär unserer Mailingliste verwenden. Nur so ist es möglich, dass alle an die Informationen kommen und mit reden können. Keiner ist gezwungen eine bestimmte Technologie / Plattform zu nutzen (außer E-Mail).

Werden Unterhaltung außerhalb der Mailingliste vorgenommen (beispielsweise Stammtisch IRC) so sollten wichtige Entscheidungen oder Themen kurz auf der Mailingliste zusammengefasst werden.

**Comment by ancho** [comment]
Ja. Ich denke so in etwa. Wenn es etwas zu einem Konkreten Issue beiträgt, spricht nichts gegen die Nutzung der Kommentare. 

Wenn jemand eine bessere Lösung hat, dann immer her damit :)

Anders herum kann es auch Sinnvoll sein, das man Ideen oder Dinge aus Diskussionen die per Mail beigesteuert werden auch wieder einem Issue hinzugefügt. So kann man auch schön nachvollziehen wie es bei einer Lösung zu einer bestimmten Entscheidung gekommen ist.
